src/Features/CSharp/Portable/LanguageServices/CSharpSymbolDisplayService.SymbolDescriptionBuilder.cs

```diff
@@ -197,6 +197,31 @@ protected override void AddCaptures(ISymbol symbol)
                 }
             }
 
+            protected override IEnumerable<SymbolDisplayPart> FieldDescriptionModifiers(IFieldSymbol symbol)
+            {
+                var desc = Enumerable.Empty<SymbolDisplayPart>();
+
+                if (symbol.IsStatic)
+                {
+                    desc = desc.Concat(Keyword(SyntaxFacts.GetText(SyntaxKind.StaticKeyword)))
+                        .Concat(Space());
```

Comment:
1. in general, localized sections of text shoudl not be created by direct concatting of strings.
2. as an enumerable, it's pretty hinky to just keep concatting here.  You can just make an ArrayBuilder, add what you want to it, and return the results at teh end.

Reply:
That makes sense - I'll update that now. Thanks!